### PR TITLE
Heavy-ion jet sequences for pp_on_AA_2018 era

### PR DESCRIPTION
**Pull request by stepobr**
This pull request is for having a proper set of heavy-ion specific jet algorithms within pp_on_AA_2018 era. There is also a fix to make the centrality producer run in this era.
@mverwe @mandrenguyen @cfmcginn @FHead

### CODE REVIEW

**Comment by slava77** [comment]
@stepobr 
there is a merge conflict in  ```Configuration/StandardSequences/python/Reconstruction_cff.py ```
please check and update

**Comment by slava77** [comment]
@cmsbuild please test workflow 158.0

**Comment by slava77** [comment]
@cmsbuild please test workflow 158.0

**Comment by perrotta** [comment]
```
ValueError: Trying to override definition of PFTowers while it is used by the sequence reconstructionHeavyIons_HcalNZS
 new object defined in: /build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_10_3_X_2018-08-21-1100/python/RecoJets/JetProducers/HiRecoPFJets_cff.py
 existing object defined in: /cvmfs/cms-ib.cern.ch/nweek-02538/slc6_amd64_gcc700/cms/cmssw/CMSSW_10_3_X_2018-08-21-1100/python/RecoHI/HiJetAlgos/HiRecoPFJets_cff.py
```

There are several unneeded duplications in your configs: try to avoid it

File: Configuration/StandardSequences/python/Reconstruction_cff.py [hunk 1]
```diff
@@ -159,6 +159,7 @@
 _highlevelreco_HI += hiClusterCompatibility
 (pp_on_XeXe_2017 | pp_on_AA_2018).toReplaceWith(highlevelreco, _highlevelreco_HI)
 
+
```
Comment (perrotta) [inline Configuration/StandardSequences/python/Reconstruction_cff.py]: Remove this newly added empty line

**Comment by stepobr** [comment]
Thanks @perrotta, @slava77, I will close the pull request and create a new one, which is better organized and without any duplication.

**Pull request closed without merging**